packages/sdks/vue-sdk/src/hooks.ts

```diff
@@ -21,7 +21,7 @@ export const useSession = () => {
   const { session } = injectDescope();
 
   if (session.isFetchSessionWasNeverCalled.value) {
-    session.fetchSession();
+    session.fetchSession(true);
```

Comment:
Are we always using try refresh in the framework SDKs?
Don't we want the descoper to configure it?

Reply:
why should they want/need to configure?

I don't think this is a Descoper decision rather to use the `refresh` or the `try-refresh` api
in the context of the SDK behavior they are the same, its just that when token is missing/invalid/expire - one return 401 and the other 200